andela/travel_tool_front
# #162701432 Create residence content loaders

#### What does this PR do?
create placeholders to represent the expected data appearance on residence

#### Description of Task to be completed?
- Replace the spinner on the residence with content placeholder loaders 

#### How should this be manually tested?
- Clone the repo `https://github.com/andela/travel_tool_front.git`
- Checkout to the branch:
`ch-residence-content-loaders-162701432Install dependencies:
```yarn install```
`yarn test`
- All the tests should be passing
- Start the application:
```yarn start```

- Login to the application
- Navigate to the dashboard page on the sidebar
- You should be able to get results as shown in the screenshot below.

#### Any background context you want to provide?
2. You should have the user role of a Travel Administrator or Super Administrator to be able to see the `manage` page. To do that:
  - Download `Postico` and connect to your travela database with relevant details: Db name, password, port, user
  - Click `userRoles` table and set the `roleId` to that of a Travel Administrator(29187) or Super[]Administrator(10948) 

#### What are the relevant pivotal tracker stories?
[#162701432](https://www.pivotaltracker.com/story/show/162701432)

#### Screenshots (if appropriate)
![ezgif com-video-to-gif 3](https://user-images.githubusercontent.com/22371596/50094183-9fd29600-0223-11e9-94a7-a65724e1e704.gif)
